### ISSUE TITLE
Some requests in the tool view return 404

### ISSUE THREAD

**Issue by Woeler** [guratr/nova-command-runner#3]
Hello,

I followed the readme and installed everything the way described, however, the view is still empty. Am I doing something wrong here?

![image](https://user-images.githubusercontent.com/18422096/53164528-ab9a5a00-35d0-11e9-8155-6fcc55893fb0.png)

![image](https://user-images.githubusercontent.com/18422096/53164555-be149380-35d0-11e9-8db8-ec31f3568c85.png)

![image](https://user-images.githubusercontent.com/18422096/53164498-9c1b1100-35d0-11e9-989d-d3d14e9db8f7.png)

**Comment by guratr** [comment]
Hello. Maybe you didn't run 
php artisan route:cache
php artisan config:cache
after installing package / deploying ?

**Comment by Woeler** [comment]
Nope, did all of that, stil returning 404. Also tried manually adding the routes, even manually adding the routes the inspector gives out. 404 every time. I'm running nova on the root domain, if that matters.

**Comment by adrian-schnell** [comment]
> Nope, did all of that, stil returning 404. Also tried manually adding the routes, even manually adding the routes the inspector gives out. 404 every time. I'm running nova on the root domain, if that matters.

same issue here - locally it's working but not on the server

> GET https://domain.de/nova-vendor/guratr/command-runner/commands/ 404
> GET https://domain.de/nova-vendor/guratr/command-runner/history/ 404

**Comment by srinathreddydudi** [comment]
Hey @Woeler  
If you are still looking for a fix for this, I released an updated version based on this package. Please check https://github.com/binarybuilds/nova-advanced-command-runner

@guratr I felt like this package is not actively maintained. So i created a separate package based on this and added proper credits to this package.

**Comment by Woeler** [comment]
Thanks @srinathreddydudi !
